#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 9b18dd62238b7de01a58b8099733ce2e06c5f897 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Fri, 9 Dec 2022 12:12:59 +0100
Subject: [PATCH 3/4] Add group configuration to help menu

This was otherwise only reachable from the group menu and
according to feedback not easily discoverable.
---
 src/mainwindow.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/mainwindow.cpp b/src/mainwindow.cpp
index 5ab12455..1a8a87b5 100644
--- a/src/mainwindow.cpp
+++ b/src/mainwindow.cpp
@@ -624,6 +624,12 @@ void MainWindow::Private::setupActions()
             QStringLiteral("../share/doc/gnupg-vsd"));
     coll->addAction(QStringLiteral("help_doc_smartcard"), smartcard);

+    const auto groups = new DocAction(QIcon::fromTheme(QStringLiteral("dvipdf")), i18n("Group configuration"),
+            i18nc("Only available in German and English. Leave to English for other languages.",
+                "handout_group-feature_gnupg_en.pdf"),
+            QStringLiteral("../share/doc/gnupg-vsd"));
+    coll->addAction(QStringLiteral("help_doc_groups"), groups);
+
     const auto man_gpg = new DocAction(QIcon::fromTheme(QStringLiteral("help-contextual")), i18n("GnuPG Manual"),
             QStringLiteral("gnupg.pdf"), QStringLiteral("../share/doc/gnupg"));
     coll->addAction(QStringLiteral("help_doc_gnupg"), man_gpg);
diff --git a/src/kleopatra.rc b/src/kleopatra.rc
index 17b20f71..5bba1cba 100644
--- a/src/kleopatra.rc
+++ b/src/kleopatra.rc
@@ -1,5 +1,5 @@
 <!DOCTYPE gui >
-<gui name="kleopatra" version="511" >
+<gui name="kleopatra" version="512" >
     <MenuBar>
         <Menu name="file">
             <text>&amp;File</text>
@@ -96,6 +96,7 @@
             <Action name="help_doc_smartcard"/>
             <Action name="help_doc_vsa10573"/>
             <Action name="help_doc_vsa10584"/>
+            <Action name="help_doc_groups"/>
             <Separator/>
             <Action name="help_check_updates"/>
         </Menu>
--
2.38.1
